ledger/src/blockstore_processor.rs

```diff
@@ -1176,16 +1192,25 @@ pub struct ThreadExecuteTimings {
 }
 
 impl ThreadExecuteTimings {
-    pub fn report_stats(&self, slot: Slot) {
+    pub fn report_stats(&self, slot: Slot, is_unified_scheduler_enabled: bool) {
+        let (total_thread_us, total_transactions_executed) = if is_unified_scheduler_enabled {
+            (None, None)
+        } else {
+            (
+                Some(self.total_thread_us as i64),
+                Some(self.total_transactions_executed as i64),
+            )
+        };
+
         lazy! {
             datapoint_info!(
                 "replay-slot-end-to-end-stats",
                 ("slot", slot as i64, i64),
-                ("total_thread_us", self.total_thread_us as i64, i64),
-                ("total_transactions_executed", self.total_transactions_executed as i64, i64),
+                ("total_thread_us", total_thread_us, Option<i64>),
+                ("total_transactions_executed", total_transactions_executed, Option<i64>),
```

Comment:
kudos to @jbiseda for  `Option<_>` introduced by https://github.com/solana-labs/solana/pull/31975 :)